mantidproject/mantid
# Enable more SNSPowderReductions system tests

Description of work.

**To test:**

Just run `systemtest -R SumFilesTest --archivesearch` and see that there  are no failures.

*There is no associated issue.*

*Does not need to be in the release notes* because it is just updating tests.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
